org-cyf-itd/content/blocks/step-1/instructions/index.md

```diff
@@ -54,7 +54,7 @@ Software for Lawyers Ltd makes case management software. Their software helps la
 2. Ask the AI to help you write a cover letter. Give it the description of Jane and the job description provided. It may take some experimentation to get the right prompt. That’s OK: getting AI to do what you want is all about trial and error.
 3. Think about [the goals of a cover letter](https://hbr.org/2014/02/how-to-write-a-cover-letter). Read and reflect on the cover letter. Has the AI provided good links between Jane's experience and the job's requirements (e.g. does it note that she has experience in the legal sector)? How has the AI demonstrated that Jane has the right skills for the job? Has the AI added anything that wasn’t in the description of Jane? Could you do better?
 4. Think about your own future job hunt. What have you learned that will help you apply for jobs in the future? Will you use ChatGPT to help you generate a cover letter? If so, how will you make sure it's a good cover letter? If not, why not?
-5. Create a Google doc with the text of the cover letter you generated
+5. Create a Google doc with the text of the cover letter you generated. If this platform is new to you, a short guide on how to use it can be found [here](https://support.google.com/docs/answer/7068618?hl=en-GB&co=GENIE.Platform%3DDesktop).
```

Comment:
This link looks like a useful one, and I agree it would be helpful to include - thanks for the suggestion!

We generally try to avoid using "here" as link text for accessibility reasons (see https://granicus.com/blog/why-click-here-links-are-bad/) - can you try to rephrase this so that the link text describes the link? E.g. `"If this platform is new to you, you can read [this short guide on Google Docs](...)."`

Reply:
Thanks for the feedback. Just pushed with amended phrasing :) 

Won't be using "here" again! Had never considered its impact before.